src/store/actions.ts

```diff
@@ -268,6 +267,8 @@ const commitExistingData = ({ commit, getters }): void => {
   }
   if (getters.getNr.corpNum) {
     commit('mutateCorpNum', getters.getNr.corpNum)
+  } else {
+    commit('mutateCorpNum', '')
```

Comment:
I don't think the purpose of this action ("commitExistingData") is to clear fields in the newRequestModel object, so I don't understand why you're doing this here.

Reply:
See comments above. Resolving.